.github/resources/codeql-config.yml

```diff
@@ -0,0 +1,10 @@
+name: CodeQL Config
+
+queries:
+  - uses: security-and-quality
+
+paths:
+  - src
+paths-ignore:
+  - src/node_modules

```

Comment:
Is this path correct? I wouldn't expect "node_modules" to exist inside the "src" folder.

Reply:
Before:
![image](https://user-images.githubusercontent.com/37381190/129207736-6c4e3826-d5e8-4225-8805-8b3265c68da5.png)
After:
![image](https://user-images.githubusercontent.com/37381190/129208925-793a8c26-8189-4409-adba-2d706a998bb3.png)


Seems like nothings changed 🤔